recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
     number: 0
-    skip: True  # [py35]
+    skip: True  # [py3k]
```

Comment:
Awesome, thanks.

Good point. If you have time, feel free to add a PR against the [doc repo]( https://github.com/conda/conda-docs ).